e2e/basics.spec.js

```diff
@@ -228,43 +228,41 @@ describe('Basic React Native and Touchable Support', () => {
         'AppContainer;|App;|Provider;|HeapNavigationWrapper;|NavigationContainer;|Navigator;|NavigationView;|TabNavigationView;|ScreenContainer;|ResourceSavingScene;|SceneView;|Connect(BasicsPage);|BasicsPage;|TouchableOpacity;[testID=touchableOpacityText];|';
       const expectedTargetText = 'Touchable Opacity Foo';
       await rnTestUtil.assertAutotrackHierarchy(
-        'touchableHandlePress',
-        expectedHierarchy,
-        expectedTargetText
-      );
+      'touchableHandlePress',
```

Comment:
nit: spacing seems off here.

Reply:
With `prettier`, we should never have to worry about that thing.  If we want something different, we need to build it into the formatter rules!

Reply:
Yea, I'm also not a huge fan of our current prettier formatting rules, but it's better than inconsistent formatting and having to manually format everything